File: examples/check_dir_existence/check_dir_existence.rb
base_dir = File.dirname(__FILE__)
exit_dir = base_dir + '/a_dir'
not_exist_dir = base_dir + '/b_dir'
if(File.exist?(exit_dir)) 
  puts("Found: " + exit_dir)
end

if(!File.exist?(not_exist_dir)) 
  puts("Not Found: " + not_exist_dir);
end

File: examples/check_dir_existence/check_dir_existence.js
const fs = require('fs')
base_dir = __dirname
const exist_dir = base_dir + '/a_dir'
const not_exist_dir = base_dir + '/b_dir'
if(fs.existsSync(exist_dir)) { 
  console.log("Found: " + exist_dir)
}
if(!fs.existsSync(not_exist_dir)) { 
  console.log("Not Found: " + not_exist_dir)
}